### DISCUSSION TITLE
[Feature] Direct upload to shared album from device in app

### DISCUSSION THREAD

**Discussion by daniel-wassmer** [immich-app/immich#10959] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

In the browser (both PC and Phone) you can make a direct upload from the device to a shared album. The app doesn't support that. Is that a bug or just forgotten or is there a reason to this. Would appreciate that, so we can use immich for sharing the vacation photos with our friends.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by echondropoulos** [comment]
(copying and pasting my comment from a closed FR)

This is a must have feature for myself. I share a private Immich instance with about a dozen family members and many refuse to use the app due to not being able to upload photos directly, or they simply do not understand how to upload a custom folder/album without uploading their entire phone.

Frankly I am surprised this isn't already available within the app.

**Comment by bo0tzz** [comment]
#1678

**Comment by timonrieger** [comment]
dupe of #12190, please upvote instead of commenting there. Thanks!
